css.md

```diff
@@ -333,4 +333,21 @@ When using media queries we can adopt the following approaches
     ```
 
 
+### 12. One method for permanently fixing the footer at the bottom. 
```

Comment:
This is a very specific use-case, does not belong in Cheat Sheets, please wite this as a post instead

Reply:
> This is a very specific use-case, does not belong in Cheat Sheets, please wite this as a post instead
where can I post this one in the repo? guide please!

Reply:
Posts are not powered by GitHub, they are part of https://onecompiler.com/